### DISCUSSION TITLE
iDrive6 no apps checkbox with connecteddrive essentials

### DISCUSSION THREAD

**Discussion by doublebigmak** [BimmerGestalt/AAIdrive#675] [Q&A] [answered]
Hey everyone,

I've come to realize that I don't have the apps checkbox when I start up my car. I've read that you need a connected drive subscription (and I do have the essentials package for the next couple of years or so - basically I'm able to connect my BMW and use remote services etc.) Do I need the full connected drive subscription for apps to show up or is there a work around?

Thanks

**Comment by hufman** [comment, marked as answer]
I am not sure about the subscription levels, which are different in different countries and hard to find documentation for. I understood the remote services is usually bundled with the same functionality as BMW Apps. Have you just recently purchased the subscription? The car usually takes a day or two to realize that the subscription has changed.
Otherwise, I'm not sure.

**Reply by doublebigmak** [reply]
Ah reading from some other sources, seems like I do need the full connecteddrive subscription to get this working, at least for the new My BMW and idrive 6... will test it out and see if that works

**Reply by doublebigmak** [reply]
Ok so for whoever's looking at this thread in the future, if you're Canadian with two levels of ConnectedDrive subscription - you need to subscribe to the full package (with weather, RTTI etc.) to get this app working.

**Reply by hufman** [reply]
Interesting, thank you for the information! I'm surprised the remote access is cheaper than the weather and traffic info!

**Reply by galenzhao** [reply]
The app checkbox only appeared If you at least have a working weather function on idrive.
The very base SOS subscription is not enough.
